js/battle.js

```diff
@@ -3219,7 +3219,16 @@ var Battle = (function () {
 			if (kwargs.prepare || kwargs.anim === 'prepare') {
 				this.prepareMove(pokemon, move, target);
 			} else if (!kwargs.notarget) {
-				(kwargs.anim ? Tools.getMove(kwargs.anim) : move).anim(this, [pokemon.sprite, target.sprite]);
+				move = kwargs.anim ? Tools.getMove(kwargs.anim) : move;
+				if (target.side.active.length > 1 && (move.target === 'allAdjacent' || move.target === 'allAdjacentFoes')) {
```

Comment:
No, that is the least correct way to do it in the history of ways to do it.

Do it in scripts.js where the spread/non-spread-ness is read.